src/utilities/codemirror/command-info-mapper.ts

```diff
@@ -0,0 +1,30 @@
+import type { SyntaxNode } from '@lezer/common';
```

Comment:
Can we keep the naming convention of new files to camelCase? I know it's not stated anywhere in the docs, but I was going to add that if we ever did a refactoring sprint.

Reply:
Also, because this is a typings file, can we move this to the `/types` folder?

Reply:
sure, I'll change it on this PR, it's a new file so it's easiest now and I introduced multiple-new-files.ts

Reply:
Thanks! I was hoping that would be the case.